src/Components/PresentationalComponents/TableColumns/RemediationColumn.js

```diff
@@ -0,0 +1,43 @@
+import React, { Fragment } from 'react';
+import PropType from 'prop-types';
+import { AnsibeTowerIcon } from '@patternfly/react-icons';import { Tooltip } from '@patternfly/react-core';
+import { FormattedMessage } from 'react-intl';
+import messages from '../../../Messages';
+import { OutlinedQuestionCircleIcon } from '@patternfly/react-icons';
+
+const RemediationColumn = ({ fixable }) => {
+    return (
+        <Fragment>
+            {fixable === true && <Fragment><AnsibeTowerIcon/> <FormattedMessage {...messages.playbook} /></Fragment>}
+            {fixable === false && <FormattedMessage {...messages.manual} />}
+            {
+
+                (fixable === null || fixable === undefined) &&
```

Comment:
I believe `fixable ?? ...` is equivalent to this line.

> The nullish coalescing operator (??) is a logical operator that returns its right-hand side operand when its left-hand side operand is null or undefined, and otherwise returns its left-hand side operand.

Reply:
:heart_eyes:  thanks for the reminder good catch. I will update it

Reply:
done